resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -519,6 +519,21 @@ spec:
                 type: int
               - name: policy_violations
                 type: int
+      - name: reports
+        type: reports
+        params:
+          - filePattern: "whitesource/**"
+            type: whitesource-ip
+          - filePattern: "whitesource-riskReport.pdf"
+            type: whitesource-ip
+          - filePattern: "**/toolrun_whitesource_*.json"
+            type: whitesource-ip
+          - filePattern: "whitesource/**"
```

Comment:
Do we upload things twice if whitesource-ip and whitesource-security are 100% overlap or will it be consolidated to one?